src/utils/slpFileWriter.ts

```diff
@@ -8,18 +6,20 @@ import { SlpFile } from "./slpFile";
 import type { SlpRawEventPayload, SlpStreamSettings } from "./slpStream";
 import { SlpStream, SlpStreamEvent } from "./slpStream";
 
+import { format } from "date-fns";
+
 /**
  * The default function to use for generating new SLP files.
  */
-function getNewFilePath(folder: string, m: Moment): string {
-  return path.join(folder, `Game_${m.format("YYYYMMDD")}T${m.format("HHmmss")}.slp`);
+function getNewFilePath(folder: string, d: Date): string {
+  return path.join(folder, `Game_${format(d, "yyyymmdd")}T${format(d, "HHmmss")}.slp`);
```

Comment:
```suggestion
  return path.join(folder, `Game_${format(d, "yyyyMMdd")}T${format(d, "HHmmss")}.slp`);
```
Make sure we don't confuse month and minutes.

Reply:
Right